src/main.c

```diff
@@ -164,7 +165,12 @@ int main(int argc, char **argv)
 	}
 
 	gettimeofday(&start,NULL);
-	handle_svf_command(jtag_handler, svf_path);
+	ret = handle_svf_command(jtag_handler, svf_path);
+	fprintf(stderr, "handle_svf_command %d\n", ret);
```

Comment:
Is this print necessary?

Reply:
This print helps to check the error code ID returned by the handle_svf_command.